### ISSUE TITLE
[BUG] Something went horribly wrong with skript

### ISSUE THREAD

**Issue by RealHypnoticOcelot** [ShaneBeee/SkBee#243]
**Describe the bug**
The game hangs when certain code is loaded into the server.

**To Reproduce**
The following lines induce this error:
```
set {_blindpot} to splash potion with nbt compound from "{CustomPotionColor:6381921,CustomPotionEffects:[{Amplifier:2b,Duration:60,Id:15}],Potion:"minecraft:empty"}"
register new shaped recipe for {_blindpot} using emerald, eye of ender, emerald, eye of ender, glass bottle, eye of ender, emerald, eye of ender, and emerald with id "splash_potion"
```

**Expected behavior**
A blindness splash potion which gave three seconds of Blindness III would be craftable.

**Screenshots/Errors**
A large paste of the errors it threw
https://pastebin.com/raw/gpmyXMHr

**Version Info:**
```
17.08 18:54:19 [Server] INFO --- [SkBee Loading Info] ---
17.08 18:54:19 [Server] INFO - [NBTAPI] Found Spigot: v1_19_R1! Trying to find NMS support
17.08 18:54:19 [Server] INFO - [NBTAPI] NMS support 'MC1_19_R1' loaded!
17.08 18:54:19 [Server] INFO - NBT Elements successfully loaded
17.08 18:54:19 [Server] INFO - Recipe Elements successfully loaded
17.08 18:54:19 [Server] INFO - Scoreboard Elements successfully loaded
17.08 18:54:19 [Server] INFO - Team Elements successfully loaded
17.08 18:54:19 [Server] INFO - Bound Elements successfully loaded
17.08 18:54:19 [Server] INFO - Text Component Elements successfully loaded
17.08 18:54:19 [Server] INFO - Pathfinding Elements successfully loaded
17.08 18:54:19 [Server] INFO - Structure Elements successfully loaded
17.08 18:54:19 [Server] INFO - Other Elements successfully loaded
17.08 18:54:19 [Server] INFO - Virtual Furnace Elements disabled via config
17.08 18:54:19 [Server] INFO - World Creator Elements successfully loaded
17.08 18:54:19 [Server] INFO - Game Event Elements successfully loaded
17.08 18:54:19 [Server] INFO - BossBar Elements successfully loaded
17.08 18:54:19 [Server] INFO - Statistic Elements successfully loaded
17.08 18:54:19 [Server] INFO - Villager Elements successfully loaded
17.08 18:54:19 [Server] INFO - Advancement Elements successfully loaded
17.08 18:54:19 [Server] INFO - World Border Elements successfully loaded
17.08 18:54:19 [Server] INFO --- [Server Info] ---
17.08 18:54:19 [Server] INFO Server Version: git-Paper-"86f87ba" (MC: 1.19)
17.08 18:54:19 [Server] INFO Skript Version: 2.6.3
17.08 18:54:19 [Server] INFO Skript Addons:
17.08 18:54:19 [Server] INFO - skript-placeholders v1.5.2
17.08 18:54:19 [Server] INFO - skRayFall v1.9.28
17.08 18:54:19 [Server] INFO SkBee Version: 2.0.0
17.08 18:54:19 [Server] INFO SkBee Website: https://github.com/ShaneBeee/SkBee
```
**Additional context**
it looks like the issue managed to hide the scoreboard objectives from showing on tablist and belowname

**Comment by ShaneBeee** [comment]
those lines have nothing to do with the error
the error states
```
Current item: discover recipe[s] "*" for all entities of type [[entitydata:player]]
```

**Comment by ShaneBeee** [comment]
Running your code above I get 2 issues:
1) the NBT line is invalid
2) once I fixed that, I too get a hang in the server. But this is not actually an SkBee issue, this is a Skript issue, where Skript doesn't like long lines.

In cases of parsing issues, I recommend breaking up the lines, ex:
```vb
set {_eye} to eye of ender
set {_em} to emerald
register new shaped recipe for {_blindpot} using {_em}, {_eye}, {_em}, {_eye}, glass bottle, {_eye}, {_em}, {_eye}, and {_em} with id "splash_potion"

```
I did this, and parsing times changed greatly
before: `[Skript] Successfully reloaded test.sk. (46275ms)`
after: `[Skript] Successfully reloaded test.sk. (15ms)`

Closing this as this is not an SkBee issue
